pulsar-discovery-service/src/main/java/org/apache/pulsar/discovery/service/ServerConnection.java

```diff
@@ -101,7 +102,7 @@ protected void handleConnect(CommandConnect connect) {
                 return;
             }
         }
-        ctx.writeAndFlush(Commands.newConnected(connect.getProtocolVersion()));
+        ctx.writeAndFlush(Commands.newConnected(connect.getProtocolVersion(), ServiceConfig.MAX_MESSAGE_SIZE));
```

Comment:
`Commands.DEFAULT_MAX_MESSAGE_SIZE`

Reply:
I think discovery service only for looking up service so it's doesn't need set max message size, right?